bin/aws-cli-mfa-ksh

```diff
@@ -65,7 +65,7 @@ def parse_cli_args():
     argparser.add_argument('-x', '--no-export-profile', action='store_true',
         help='do not export/set AWS_PROFILE to the one generated by STS')
     argparser.add_argument('-d', '--lifetime-duration', type=int,
-        help='do not export/set AWS_PROFILE to the one generated by STS')
+        help='set the time, in seconds, that the access should last (default 12 hours)')
```

Comment:
Oh, yeah, this was me -- I forgot to run build after changing this: https://github.com/rogusdev/aws-cli-mfa/commit/2ba000ad47ec686c5d6d68ff458fbc3f8aad2d84 -- so I can fix that separately from your PR in a bit

Reply:
You need to rebase/merge now -- I updated these correctly on main